src/components/FeedbackPage/FeedbackPage.js

```diff
@@ -4,31 +4,36 @@ import {
   LINKS,
   ELEMENT_IDS,
 } from '../../constants';
+import { rememberVisitToFeedbackPage } from '../../helpers/userBehaviour';
 import style from './FeedbackPage.scss';
 
-const FeedbackPage = () => (
-  <PageWrapper dimFeedback>
-    <div className={style.feedbackWrapper}>
-      <div className={style.container}>
-        <h1
-          id={ELEMENT_IDS.INITIAL_FOCUS_POINT_ON_PAGE}
-          className={style.title}
-        >Please give feedback</h1>
+const FeedbackPage = () => {
+  rememberVisitToFeedbackPage();
 
-        <p className={style.paragraph}>
-          The Smart Cities Performance Dashboard is a working prototype, and we value your
-          feedback and suggestions.
-        </p>
+  return (
+    <PageWrapper dimFeedback>
+      <div className={style.feedbackWrapper}>
+        <div className={style.container}>
+          <h1
+            id={ELEMENT_IDS.INITIAL_FOCUS_POINT_ON_PAGE}
+            className={style.title}
+          >Please give feedback</h1>
 
-        <iframe
-          className={style.surveyFrame}
-          title="feedback"
-          allowTransparency="true"
-          src={LINKS.FEEDBACK_SURVEY_EMBED}
-        />
+          <p className={style.paragraph}>
+            The Smart Cities Performance Dashboard is a working prototype, and we value your
```

Comment:
"Smart Cities Performance Dashboard"?

Reply:
No idea why this is showing up as new - probably git getting confused.

Reply:
Oh, I new it wasn't knew, just commenting asking should it be changed to the snazzy new name too?

I know I got my new and knew mixed up but I'm committed now.

Reply:
I'm going thru the source code to find instances of this though....